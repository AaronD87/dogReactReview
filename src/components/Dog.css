.dogName {
  margin: 1em 0 0.5em 0;
	color: #343434;
	font-weight: normal;
	font-size: 1.75em;
	line-height: 40px;
	font-family: 'Orienta', sans-serif;
}
.dogAge {
  margin: 1em 0 0.5em 0;
	color: #343434;
	font-weight: normal;
	font-size: 1.25em;
	line-height: 40px;
	font-family: 'Orienta', sans-serif;
}
.dogWeight {
  margin: 1em 0 0.5em 0;
	color: #343434;
	font-weight: normal;
	font-size: 1.25em;
	line-height: 40px;
	font-family: 'Orienta', sans-serif;
}