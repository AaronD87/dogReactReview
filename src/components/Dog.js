import React from 'react';
import PropTypes from 'prop-types';
import styles from './Dog.css';

export default function Dog({ age, name, weight }) {
  return (
    <>
      <h2 className={styles.dogName}>{name}</h2>
      <p className={styles.dogAge}>Age: {age} years old</p>
      <p className={styles.dogWeight}>Weight: {weight}</p>
    </>
  );
}


Dog.propTypes = {
  age: PropTypes.number.isRequired,
  name: PropTypes.string.isRequired,
  weight: PropTypes.string.isRequired,
};
