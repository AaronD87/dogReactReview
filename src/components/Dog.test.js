import React from 'react';
import { shallow } from 'enzyme';
import Dog from './Dog';

describe('Dog', () => {
  it('renders the Dog component', () => {
    const wrapper = shallow(<Dog age={12} name={'Billy'} weight={'70 lbs'} />);

    expect(wrapper).toMatchSnapshot();
  });
});
