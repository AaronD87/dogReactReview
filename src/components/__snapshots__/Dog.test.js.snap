// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dog renders the Dog component 1`] = `
<Fragment>
  <h2
    className="dogName"
  >
    Billy
  </h2>
  <p
    className="dogAge"
  >
    Age: 
    12
     years old
  </p>
  <p
    className="dogWeight"
  >
    Weight: 
    70 lbs
  </p>
</Fragment>
`;
